app/src/controllers/object.js

```diff
@@ -929,14 +929,20 @@ const controller = {
         public: isTruthy(req.query.public),
         active: isTruthy(req.query.active),
         deleteMarker: isTruthy(req.query.deleteMarker),
-        latest: isTruthy(req.query.latest)
+        latest: isTruthy(req.query.latest),
+        page: req.query.page,
+        limit: req.query.limit,
+        sort: req.query.sort,
+        order: req.query.order,
+        permissions: isTruthy(req.query.permissions)
```

Comment:
Optional Style: have permissions come before the page/limit/sort/order as permissions is logical controller functionality - let's aim for pagination parameters to be conventionally the last to show up in a parameter set.